src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -925,7 +925,7 @@ bool clang_cpp_convertert::get_function_this_pointer_param(
   this_param.cmt_identifier(id);
 
   // Add to the list of params
```

Comment:
It's probably better to update the comment here. "// Add `this` as the first parameter to the list of params". 
Or just move it if you wish, the code seems pretty self-explanatory.